datashader/callbacks.py

```diff
@@ -114,10 +137,13 @@ def redraw_image(self, ranges):
         dh = y_range[1] - y_range[0]
         dw = x_range[1] - x_range[0]
 
-        image = self.callback(x_range, y_range, ranges['w'], ranges['h'], **self.kwargs)
+        image = self.callback(x_range=x_range, y_range=y_range, w=ranges['w'], h=ranges['h'], **self.kwargs)
         self.ds.data.update(dict(image=[image.img], x=[x_range[0]], y=[y_range[0]],
                                  dw=[dw], dh=[dh]))
 
 
     def _repr_html_(self):
         return self.div
+
+
+IPythonKernelCallback = InteractiveImage  # Temporary; for backwards compatibility
```

Comment:
Right; I'll be removing that shortly.  At the time I had a lot of different notebooks, but I think they are all updated now.